.github/workflows/docker_build_push.yml

```diff
@@ -38,3 +38,45 @@ jobs:
           ref: main
           wait_interval: 60
           client_payload: '{ "git_commit": "${{ github.sha }}", "docker_tag": "${{ env.docker_tag }}" }'
+
+  docker_e2e:
```

Comment:
One last comment: We could move `docker_e2e` before `k8s_e2e` and making the latter dependent on the former (with an extra `need`). This would save some CI time in case `docker_e2e` fails.

Reply:
Ah yes good idea !